proto/eth/service/key_management.proto

```diff
@@ -167,6 +167,19 @@ service KeyManagement {
       body: "*"
     };
   }
+
+  // GetGasLimit returns the execution gas limit for an individual validator.
+  //
+  // HTTP response status codes:
+  //  - 400: Bad request
+  //  - 401: Unauthorized
+  //  - 403: Forbidden
+  //  - 500: Validator internal error
+  rpc GetGasLimit(PubkeyRequest) returns (GetGasLimitResponse) {
+    option (google.api.http) = {
+      get: "/internal/eth/v1/validator/{pubkey}/gaslimit"
```

Comment:
the associated endpoint factory `./validator/rpc/apimiddleware/endpoint_factory.go` will also need to be added for this feature to work.  once this is done it should work but will require curl or postman testing

Reply:
Done, thanks.